tests/unit/plugin.spec.js

```diff
@@ -24,68 +24,31 @@ describe("CLD plugin", () => {
       const localVue = createLocalVue();
       localVue.use(Cloudinary, { components: [] });
 
-      const wrapper = mount(
-        {
-          template: `<cld-image cloudName="demo" publicId="face_top" />`
-        },
-        { localVue }
-      );
-      expect(wrapper.is("img")).toBe(false);
+      expect(Object.hasOwnProperty.call(localVue.options.components, 'CldImage')).toBe(false);
     });
 
     it("empty object installs no component", async () => {
       const localVue = createLocalVue();
       localVue.use(Cloudinary, { components: {} });
 
-      const wrapper = mount(
-        {
-          template: `<cld-image cloudName="demo" publicId="face_top" />`
-        },
-        { localVue }
-      );
-      expect(wrapper.is("img")).toBe(false);
+      expect(Object.hasOwnProperty.call(localVue.options.components, 'CldImage')).toBe(false);
```

Comment:
Is it possible to write like this instead?
`localVue.options.components.hasOwnProperty('CldImage')` ? 
Or was there a reason for `Object....cal()?`

Also, what do you think about making the expect a bit clearer as well?
```
let isPluginComponentDefined = localVue.options.components.hasOwnProperty('CldImage');
expect(isPluginComponentDefined).toBe(true);
```

Reply:
Such invocation of `hasOwnProperty` method is following eslint recommended rule [no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins), but is not essential.

I agree that readability of `expect` statement can be improved. Will push updates to current branch.

Reply:
Looks good, @mayashavin any further comments before we merge?